packages/minimongo/matcher.js

```diff
@@ -5,6 +5,25 @@ import {
   nothingMatcher,
 } from './common.js';
 
+let Decimal = Package['mongo-decimal']?.Decimal;
+
+// Due to {weak: true} Decimal dependency on client
+// declare minimal Decimal functionality
+if (!Decimal) {
```

Comment:
This has no value, since the instanceof checks will never be true for this class.

Reply:
For clarity: The reason I proposed a DecimalStub{} class was just to have some kind of object that doesn't throw an error when we do instanceof, but which would never be true. If someone wants to have decimal support, they should add the mongo-decimal package.

Reply:
Thanks for help! I was change code with your recommendations.